### ISSUE TITLE
Grunt task complains about !important in styles.css

### ISSUE THREAD

**Issue by revosw** [KQMATH/moodle-mod_jazzquiz#67]
When grunt runs the `stylelint:css` task, it complains about these lines
https://github.com/KQMATH/moodle-mod_jazzquiz/blob/dcaa2ff7f09f54d2e499c6f5e2aebdd62f74d190/styles.css#L461-L464

**Comment by revosw** [comment]
Is a css preprocessor like sass/less or anything involved in producing this file?

**Comment by andstor** [comment]
No, that file is handwritten. If the `!important` is actually necessarily I don't know. If the CI complains about it, it might be favorable to check if the rule can be mitigated. Alternatively, the warning could be depressed.

**Comment by hgeorgsch** [comment]
@sebastsg Do you have a quick comment on !important in JazzQuiz CSS?

My guess is that it has been too hard to narrow down the tag/class definition to override what should be overridden, and !important provides a quick fix.

**Comment by sebastsg** [comment]
@hgeorgsch I never add `!important` for fun, so I'm pretty sure it's to override MathJax. I'll take a look soon.

**Comment by hgeorgsch** [comment]
I thought it should be possible to get it right with a narrower class definition, just bloody hard; that has been my experience so far. But yes, since the last class is MathJax_Display, I am sure that is the reason.

Oddly, when I check the source of an actual JazzQuiz page, I find only one or two CSS files and they do not show any other definitions for MathJax_Display.  May this be out of date?  Or is it a tag attribute style which has to be overridden?  I did not have the chance to test with a full student view.

**Comment by jorgenfinsveen** [comment]
Tried removing !important from the CSS and reload the page. Did not seem to affect the style, so I don't believe we need to use !important anymore. Tested by comparing two containers where one container hosted Jazzquiz with the !important statement in the CSS, and one without. They were identical (tested this at different resolutions as well).

<br/><br/>

Screenshots from pages which uses the CSS statements in question:

<br/>

**The list on the left is target for the CSS block. It doesn't seem to be affected after removing !important**
<img width="1351" alt="Skjermbilde 2023-07-31 kl  15 33 27" src="https://github.com/KQMATH/moodle-mod_jazzquiz/assets/44703456/e25c729e-3f2e-451a-9fad-9425e6fec8e0">

<br/><br/>

**Dropdown-menu is also target for the CSS, but has not changed after removing !important**
<img width="604" alt="Skjermbilde 2023-07-31 kl  16 07 31" src="https://github.com/KQMATH/moodle-mod_jazzquiz/assets/44703456/5462f8f3-a20f-42d8-a394-4a6a25fe17ad">


<br/>

I also looked for elements of the MathJax_Display class by using inspect element, but I could not find any mentions of the class anywhere. Also checked the source code of Jazzquiz and the source code of Moodle version 4.1 and version 4.3, and the only place where the phrase MathJax_Display is mentioned is in this CSS file. As @hgeorgsch mentioned, it is likely that MathJax_Display is outdated.


All in all, I don't think we need !important in the CSS anymore.
